optimism.tokenlist.json

```diff
@@ -1525,6 +1525,17 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 42,
+      "address": "0xEf2E078a649aAd3908b7F4f9aD75A881D4f3b7e3",
+      "name": "Biconomy Token",
+      "symbol": "BICO",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BICO.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
```

Comment:
Please add the information for the token on Optimism Kovan. Thanks!

Reply:
Done